packages/hoprd/environments.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Documentation of the entries is still missing. Since the file combines environments and networks I would use a neutral name for the filename instead.

Reply:
Any naming suggestions? 

What is environment? What is network? How is this file called?

Reply:
`protocol-config.json`